src/test/java/org/broadinstitute/hellbender/tools/exome/NormalizeBySampleDepthIntegrationTest.java

```diff
@@ -0,0 +1,108 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.apache.commons.math3.linear.ArrayRealVector;
+import org.apache.commons.math3.linear.RealMatrix;
+import org.apache.commons.math3.linear.RealVector;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Integration tests for {@link CombineReadCounts}.
+ *
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+public class NormalizeBySampleDepthIntegrationTest extends CommandLineProgramTest {
```

Comment:
When you separate out the backend class, consider writing automated test(s) for it.